### PR TITLE
[Sponsored by Holybro] Add Goermicro/Goertek SPA06 baro driver

### PR DESCRIPTION
**Pull request by julianoes**
This adds a driver for the Goermicro/Goertek SPA06, mostly copying the SPL06 driver.

So far this is only bench tested, over I2C.

FYI @vincentpoont2 

@dagar let me know if you want this as a variant of the SPL06 instead and call it SPx06.

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rc.sensors [hunk 1]
```diff
@@ -211,6 +211,13 @@ then
 	spl06 -X -a 0x77 start
 fi
 
+# SPA06 sensor external I2C
+if param compare -s SENS_EN_SPA06 1
+then
+	spa06 -X start
+	spa06 -X -a 0x77 start
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/rc.sensors:218]: Why handle 0x77 here?

**Comment by dagar** [comment]
This isn't included in any builds?

**Comment by julianoes** [comment]
@dagar it's in the Kakutes, see third commit.

File: ROMFS/px4fmu_common/init.d/rc.sensors (hunk 1 above)
Comment (julianoes) [reply ROMFS/px4fmu_common/init.d/rc.sensors:218]: Just copied SPL06. Would you do it somewhere else?

Comment (julianoes) [reply ROMFS/px4fmu_common/init.d/rc.sensors:218]: @dagar let me know what you want me to change here, otherwise I'll merge as is.

**Comment by vincentpoont2** [comment]
@julianoes Can we backport this?

**Pull request merged**